CONTRIBUTING.md

```diff
@@ -0,0 +1,2 @@
+#Guide to Contributing to JaaS
+[fork, clone, pull-request] (https://guides.github.com/activities/contributing-to-open-source/)
```

Comment:
Did not have to remove the formatting guide, it was just all on one line when you looked at the doc